lib/assessment.rb

```diff
@@ -43,6 +48,8 @@ def transform(words)
   # replacing the trailing newline character with a
   # space and return the result.
   def read_file(filename)
-    # Your code goes here
+    file.open("filename.txt") {|f| f.readline}
```

Comment:
However, this will still not work - the code for handling the file's contents must be found inside the block _only_. Additionally, `readline` does not look like it's a valid method for a File to call.